docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
+## Environment types
 
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
+Platform.sh offers three types of environment groups: Production, Staging, and
+Development. You can assign permissions to each user per environment so for each environment type, users always have the same set of permissions across all environments of the same type.
 
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
+For example, if you assign user1 **Admin** permissions for Development environments,  
+**Contributor** permissions for Staging environments and **Viewer** for the Production environment. User1 effectively has **Admin** permissions to all development environments, **Contributor** permissions to all staging environments and **Viewer** permissions to the Production environment.
 
-## SSH access
+A few things to consider:
 
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
+* Only one environment per project can have Production type.
+* Environments can share the same Staging and Development type.
+* After you add or remove a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy` or by clicking **Redeploy** in the management console.
 
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
 
-## Manage user permissions in the console
+| Environment permission   | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type. This role includes the Viewer permissions.|
+|Contributor               | Contributors can push code and branch all environments of this type
+|Admin                     | This role includes Viewer and Contributor permissions. Admins can change settings and execute actions on all environments of this type. To customize who can use SSH [set the access key](/configuration/app/access.html) in your platform.app.yaml|
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+## Managing users
 
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+### Add a user from the console
```

Comment:
As we agreed with the templates, this and the next section should be in tabs, not in separate sections.